src/components/comment/comment.test.ts

```diff
@@ -66,7 +66,8 @@ test(
       .toMatchInlineSnapshot(`
         form "Reply to Test author name"
           heading "Reply to Test author name" (level=4)
-          link "Markdown"
+          status
```

Comment:
@calebeby Excellent question! I made the test fail in `headed` mode and I see the following:

<img width="1312" alt="Screen Shot 2022-07-07 at 9 42 06 AM" src="https://user-images.githubusercontent.com/459757/177826497-a1239b7f-1e0e-483e-9a39-4562f04c0665.png">

This then makes me wonder if setting a default of `role="status"` is the wrong assumption. 🤔

Reply:
@calebeby Were you thinking [like this](https://github.com/cloudfour/cloudfour.com-patterns/pull/1914#discussion_r916047227)?

Reply:
Yes, I am suggesting passing in role="" or another role for the alert in the comment form component

Reply:
Ah, got it!

Okay, I need to rework the Alert twig component. Right now, if you pass in `role: ''`, it will default to `role="status"`.

Reply:
@gerardo-rodriguez From [MDN](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/Alert_Role):

> Warning: Because of its intrusive nature, the alert role must be used sparingly and only in situations where the user's immediate attention is required.

I am wondering if maybe we should not be using role=alert on the alert component for the offline banner, either. If that is the case, maybe it doesn't make sense for `role="alert"` to be the default.

Reply:
Thanks, @calebeby! Yep, good call. I'm actually using `role="status"` and not `role="alert"` for those very reasons. 😉

Reply:
Oops 😂 I missed that